### PR TITLE
Add check for non-inclusive language

### PR DESCRIPTION
**Pull request by nhosoi**
Add a check for usage of terms and language that is considered non-inclusive. We are using the woke tool for this with a wordlist that can be found at https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/config_files/woke.yml

### CODE REVIEW

**Comment by nhosoi** [comment]
Note: If https://github.com/linux-system-roles/ad_integration/pull/10/commits/d9924fafc3fe3cb8ae59b0ad485fe09771a1b147 is acceptable, we can use the action woke-action with the released `woke`.

**Comment by nhosoi** [comment]
[citest]

File: .github/workflows/woke.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# name: woke
+on:  # yamllint disable-line rule:truthy
+  - pull_request
+jobs:
+  woke:
+    name: woke
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (richm) [inline .github/workflows/woke.yml:11]:
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/woke.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# yamllint disable rule:line-length
+# name: woke
```
Comment (richm) [inline .github/workflows/woke.yml:2]: Without `name`, the status field is called `.github/workflows/woke.yml / woke (pull_request)`
Is there some reason you have `# name: woke` commented out?

Comment (nhosoi) [reply .github/workflows/woke.yml]: No, I don't think so... Anyway, I updated the top-level name as you suggested in https://github.com/linux-system-roles/crypto_policies/pull/54#issuecomment-1351675147, which is much better. Thanks!

> Check for non-inclusive language / woke (pull_request)

**Comment by nhosoi** [comment]
@richm, @spetrosi, @justin-stephenson, could you please review this pr one more time? Thanks in advance.

**Pull request merged**